CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [2.4.3]
```

Comment:
Do we really need to maintain this?

Reply:
Yes otherwise it does not get publish on pub.dev -_- The publish tool of dart check if there is a description for the current version pushed to publish here https://pub.dev/packages/kuzzle/changelog